clippy.toml

```diff
@@ -1 +1 @@
-too-many-arguments-threshold = 9
+too-many-arguments-threshold = 10
```

Comment:
Maybe it is time to convert the formatting options into a struct?

For example: https://github.com/numtide/treefmt/blob/654ecefd62fbf7ce5c5538fb6adcfc75e916f0c9/src/command/format.rs#L52

Reply:
It's getting there, but it can wait